{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "6208054e",
   "metadata": {},
   "outputs": [],
   "source": [
    "import os"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "1d0b97d4",
   "metadata": {},
   "outputs": [],
   "source": [
    "input_wqrts = [\"GTF\", \"GTF-true\", \"GTF-bucky-boot\", \"GTF-boot\"]\n",
    "REPLICATES = 20"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "720bfc55",
   "metadata": {},
   "outputs": [],
   "source": [
    "configs = [\"lower-ILS\", \"higher-ILS\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "dc9be330",
   "metadata": {},
   "outputs": [],
   "source": [
    "base_folder = \"/home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main\"\n",
    "wqfm_path = f\"{base_folder}/Scripts-2/wQFM-v1.4.jar\"\n",
    "dom_path = f\"{base_folder}/Scripts-2/generateBestWQrts.jar\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "400cbac7",
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_dom_filename(f):\n",
    "    return f.replace(\"wqrts\", \"dqrts\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "71a6f305",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "################ GTF, lower-ILS, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R1-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R1-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R1-GTF.dqrts\n",
      "################ GTF, lower-ILS, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R2-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R2-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R2-GTF.dqrts\n",
      "################ GTF, lower-ILS, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R3-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R3-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R3-GTF.dqrts\n",
      "################ GTF, lower-ILS, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R4-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R4-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R4-GTF.dqrts\n",
      "################ GTF, lower-ILS, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R5-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R5-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R5-GTF.dqrts\n",
      "################ GTF, lower-ILS, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R6-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R6-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R6-GTF.dqrts\n",
      "################ GTF, lower-ILS, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R7-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R7-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R7-GTF.dqrts\n",
      "################ GTF, lower-ILS, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R8-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R8-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R8-GTF.dqrts\n",
      "################ GTF, lower-ILS, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R9-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R9-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R9-GTF.dqrts\n",
      "################ GTF, lower-ILS, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R10-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R10-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R10-GTF.dqrts\n",
      "################ GTF, lower-ILS, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R11-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R11-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R11-GTF.dqrts\n",
      "################ GTF, lower-ILS, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R12-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R12-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R12-GTF.dqrts\n",
      "################ GTF, lower-ILS, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R13-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R13-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R13-GTF.dqrts\n",
      "################ GTF, lower-ILS, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R14-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R14-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R14-GTF.dqrts\n",
      "################ GTF, lower-ILS, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R15-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R15-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R15-GTF.dqrts\n",
      "################ GTF, lower-ILS, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R16-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R16-GTF.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R16-GTF.dqrts\n",
      "################ GTF, lower-ILS, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R17-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R17-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R17-GTF.dqrts\n",
      "################ GTF, lower-ILS, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R18-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R18-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R18-GTF.dqrts\n",
      "################ GTF, lower-ILS, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R19-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R19-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R19-GTF.dqrts\n",
      "################ GTF, lower-ILS, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R20-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R20-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R20-GTF.dqrts\n",
      "################ GTF, higher-ILS, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R1-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R1-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R1-GTF.dqrts\n",
      "################ GTF, higher-ILS, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R2-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R2-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R2-GTF.dqrts\n",
      "################ GTF, higher-ILS, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R3-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R3-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R3-GTF.dqrts\n",
      "################ GTF, higher-ILS, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R4-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R4-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R4-GTF.dqrts\n",
      "################ GTF, higher-ILS, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R5-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R5-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R5-GTF.dqrts\n",
      "################ GTF, higher-ILS, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R6-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R6-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R6-GTF.dqrts\n",
      "################ GTF, higher-ILS, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R7-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R7-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R7-GTF.dqrts\n",
      "################ GTF, higher-ILS, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R8-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R8-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R8-GTF.dqrts\n",
      "################ GTF, higher-ILS, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R9-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R9-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R9-GTF.dqrts\n",
      "################ GTF, higher-ILS, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R10-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R10-GTF.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R10-GTF.dqrts\n",
      "################ GTF, higher-ILS, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R11-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R11-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R11-GTF.dqrts\n",
      "################ GTF, higher-ILS, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R12-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R12-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R12-GTF.dqrts\n",
      "################ GTF, higher-ILS, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R13-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R13-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R13-GTF.dqrts\n",
      "################ GTF, higher-ILS, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R14-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R14-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R14-GTF.dqrts\n",
      "################ GTF, higher-ILS, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R15-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R15-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R15-GTF.dqrts\n",
      "################ GTF, higher-ILS, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R16-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R16-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R16-GTF.dqrts\n",
      "################ GTF, higher-ILS, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R17-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R17-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R17-GTF.dqrts\n",
      "################ GTF, higher-ILS, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R18-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R18-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R18-GTF.dqrts\n",
      "################ GTF, higher-ILS, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R19-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R19-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R19-GTF.dqrts\n",
      "################ GTF, higher-ILS, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R20-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R20-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R20-GTF.dqrts\n",
      "################ GTF-bucky-boot, lower-ILS, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R1-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R1-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R1-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, lower-ILS, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R2-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R2-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R2-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, lower-ILS, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R3-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R3-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R3-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, lower-ILS, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R4-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R4-GTF-bucky-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R4-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, lower-ILS, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R5-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R5-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R5-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, lower-ILS, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R6-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R6-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R6-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, lower-ILS, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R7-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R7-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R7-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, lower-ILS, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R8-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R8-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R8-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, lower-ILS, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R9-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R9-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R9-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, lower-ILS, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R10-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R10-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R10-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, lower-ILS, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R11-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R11-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R11-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, lower-ILS, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R12-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R12-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R12-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, lower-ILS, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R13-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R13-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R13-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, lower-ILS, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R14-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R14-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R14-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, lower-ILS, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R15-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R15-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R15-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, lower-ILS, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R16-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R16-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R16-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, lower-ILS, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R17-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R17-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R17-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, lower-ILS, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R18-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R18-GTF-bucky-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R18-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, lower-ILS, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R19-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R19-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R19-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, lower-ILS, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R20-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R20-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R20-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, higher-ILS, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R1-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R1-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R1-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, higher-ILS, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R2-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R2-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R2-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, higher-ILS, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R3-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R3-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R3-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, higher-ILS, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R4-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R4-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R4-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, higher-ILS, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R5-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R5-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R5-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, higher-ILS, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R6-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R6-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R6-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, higher-ILS, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R7-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R7-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R7-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, higher-ILS, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R8-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R8-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R8-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, higher-ILS, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R9-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R9-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R9-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, higher-ILS, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R10-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R10-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R10-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, higher-ILS, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R11-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R11-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R11-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, higher-ILS, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R12-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R12-GTF-bucky-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R12-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, higher-ILS, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R13-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R13-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R13-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, higher-ILS, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R14-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R14-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R14-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, higher-ILS, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R15-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R15-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R15-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, higher-ILS, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R16-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R16-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R16-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, higher-ILS, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R17-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R17-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R17-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, higher-ILS, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R18-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R18-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R18-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, higher-ILS, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R19-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R19-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R19-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, higher-ILS, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R20-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R20-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R20-GTF-bucky-boot.dqrts\n",
      "################ GTF-boot, lower-ILS, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R1-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R1-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R1-GTF-boot.dqrts\n",
      "################ GTF-boot, lower-ILS, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R2-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R2-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R2-GTF-boot.dqrts\n",
      "################ GTF-boot, lower-ILS, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R3-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R3-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R3-GTF-boot.dqrts\n",
      "################ GTF-boot, lower-ILS, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R4-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R4-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R4-GTF-boot.dqrts\n",
      "################ GTF-boot, lower-ILS, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R5-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R5-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R5-GTF-boot.dqrts\n",
      "################ GTF-boot, lower-ILS, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R6-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R6-GTF-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R6-GTF-boot.dqrts\n",
      "################ GTF-boot, lower-ILS, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R7-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R7-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R7-GTF-boot.dqrts\n",
      "################ GTF-boot, lower-ILS, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R8-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R8-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R8-GTF-boot.dqrts\n",
      "################ GTF-boot, lower-ILS, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R9-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R9-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R9-GTF-boot.dqrts\n",
      "################ GTF-boot, lower-ILS, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R10-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R10-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R10-GTF-boot.dqrts\n",
      "################ GTF-boot, lower-ILS, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R11-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R11-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R11-GTF-boot.dqrts\n",
      "################ GTF-boot, lower-ILS, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R12-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R12-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R12-GTF-boot.dqrts\n",
      "################ GTF-boot, lower-ILS, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R13-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R13-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R13-GTF-boot.dqrts\n",
      "################ GTF-boot, lower-ILS, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R14-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R14-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R14-GTF-boot.dqrts\n",
      "################ GTF-boot, lower-ILS, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R15-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R15-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R15-GTF-boot.dqrts\n",
      "################ GTF-boot, lower-ILS, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R16-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R16-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R16-GTF-boot.dqrts\n",
      "################ GTF-boot, lower-ILS, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R17-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R17-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R17-GTF-boot.dqrts\n",
      "################ GTF-boot, lower-ILS, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R18-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R18-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R18-GTF-boot.dqrts\n",
      "################ GTF-boot, lower-ILS, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R19-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R19-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R19-GTF-boot.dqrts\n",
      "################ GTF-boot, lower-ILS, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R20-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R20-GTF-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/lower-ILS/R20-GTF-boot.dqrts\n",
      "################ GTF-boot, higher-ILS, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R1-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R1-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R1-GTF-boot.dqrts\n",
      "################ GTF-boot, higher-ILS, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R2-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R2-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R2-GTF-boot.dqrts\n",
      "################ GTF-boot, higher-ILS, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R3-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R3-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R3-GTF-boot.dqrts\n",
      "################ GTF-boot, higher-ILS, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R4-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R4-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R4-GTF-boot.dqrts\n",
      "################ GTF-boot, higher-ILS, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R5-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R5-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R5-GTF-boot.dqrts\n",
      "################ GTF-boot, higher-ILS, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R6-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R6-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R6-GTF-boot.dqrts\n",
      "################ GTF-boot, higher-ILS, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R7-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R7-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R7-GTF-boot.dqrts\n",
      "################ GTF-boot, higher-ILS, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R8-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R8-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R8-GTF-boot.dqrts\n",
      "################ GTF-boot, higher-ILS, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R9-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R9-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R9-GTF-boot.dqrts\n",
      "################ GTF-boot, higher-ILS, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R10-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R10-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R10-GTF-boot.dqrts\n",
      "################ GTF-boot, higher-ILS, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R11-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R11-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R11-GTF-boot.dqrts\n",
      "################ GTF-boot, higher-ILS, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R12-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R12-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R12-GTF-boot.dqrts\n",
      "################ GTF-boot, higher-ILS, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R13-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R13-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R13-GTF-boot.dqrts\n",
      "################ GTF-boot, higher-ILS, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R14-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R14-GTF-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R14-GTF-boot.dqrts\n",
      "################ GTF-boot, higher-ILS, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R15-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R15-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R15-GTF-boot.dqrts\n",
      "################ GTF-boot, higher-ILS, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R16-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R16-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R16-GTF-boot.dqrts\n",
      "################ GTF-boot, higher-ILS, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R17-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R17-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R17-GTF-boot.dqrts\n",
      "################ GTF-boot, higher-ILS, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R18-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R18-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R18-GTF-boot.dqrts\n",
      "################ GTF-boot, higher-ILS, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R19-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R19-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R19-GTF-boot.dqrts\n",
      "################ GTF-boot, higher-ILS, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R20-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R20-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/10-taxon/Output/higher-ILS/R20-GTF-boot.dqrts\n"
     ]
    }
   ],
   "source": [
    "for input_wqrt in input_wqrts:\n",
    "    for configuration in configs:\n",
    "        if input_wqrt == \"GTF-true\":\n",
    "            continue\n",
    "#             tmp = configuration.split(\"-\")\n",
    "#             if len(tmp) > 2:\n",
    "#                 configuration = tmp[0] + \"-\" + tmp[1]\n",
    "         \n",
    "        output_folder = f'{base_folder}/10-taxon/Output/{configuration}'\n",
    "        GENES = 200\n",
    "        for replicate_num in range(1, REPLICATES+1):\n",
    "            print(f\"################ {input_wqrt}, {configuration}, Replicate {replicate_num} ########################\")\n",
    "            input_wqrts_file = os.path.join(output_folder, f\"R{replicate_num}-{input_wqrt}.wqrts\")\n",
    "            cmd = f\"java -jar {dom_path} {input_wqrts_file} {get_dom_filename(input_wqrts_file)}\"\n",
    "            print(cmd)\n",
    "            os.system(cmd)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
