{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "6208054e",
   "metadata": {},
   "outputs": [],
   "source": [
    "import os"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "1d0b97d4",
   "metadata": {},
   "outputs": [],
   "source": [
    "input_wqrts = [\"GTF\", \"GTF-true\", \"GTF-bucky-boot\", \"GTF-boot\"]\n",
    "REPLICATES = 10"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "720bfc55",
   "metadata": {},
   "outputs": [],
   "source": [
    "configs = [\"100gene-100bp\", \"100gene-1000bp\", \"1000gene-100bp\", \"1000gene-1000bp\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "dc9be330",
   "metadata": {},
   "outputs": [],
   "source": [
    "base_folder = \"/home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main\"\n",
    "wqfm_path = f\"{base_folder}/Scripts-2/wQFM-v1.4.jar\"\n",
    "dom_path = f\"{base_folder}/Scripts-2/generateBestWQrts.jar\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "400cbac7",
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_dom_filename(f):\n",
    "    return f.replace(\"wqrts\", \"dqrts\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "71a6f305",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "################ GTF, 1X-200-50, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R1-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R1-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R1-GTF.dqrts\n",
      "################ GTF, 1X-200-50, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R2-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R2-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R2-GTF.dqrts\n",
      "################ GTF, 1X-200-50, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R3-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R3-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R3-GTF.dqrts\n",
      "################ GTF, 1X-200-50, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R4-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R4-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R4-GTF.dqrts\n",
      "################ GTF, 1X-200-50, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R5-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R5-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R5-GTF.dqrts\n",
      "################ GTF, 1X-200-50, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R6-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R6-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R6-GTF.dqrts\n",
      "################ GTF, 1X-200-50, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R7-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R7-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R7-GTF.dqrts\n",
      "################ GTF, 1X-200-50, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R8-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R8-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R8-GTF.dqrts\n",
      "################ GTF, 1X-200-50, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R9-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R9-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R9-GTF.dqrts\n",
      "################ GTF, 1X-200-50, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R10-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R10-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R10-GTF.dqrts\n",
      "################ GTF, 1X-200-50, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R11-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R11-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R11-GTF.dqrts\n",
      "################ GTF, 1X-200-50, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R12-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R12-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R12-GTF.dqrts\n",
      "################ GTF, 1X-200-50, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R13-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R13-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R13-GTF.dqrts\n",
      "################ GTF, 1X-200-50, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R14-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R14-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R14-GTF.dqrts\n",
      "################ GTF, 1X-200-50, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R15-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R15-GTF.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R15-GTF.dqrts\n",
      "################ GTF, 1X-200-50, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R16-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R16-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R16-GTF.dqrts\n",
      "################ GTF, 1X-200-50, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R17-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R17-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R17-GTF.dqrts\n",
      "################ GTF, 1X-200-50, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R18-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R18-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R18-GTF.dqrts\n",
      "################ GTF, 1X-200-50, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R19-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R19-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R19-GTF.dqrts\n",
      "################ GTF, 1X-200-50, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R20-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R20-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R20-GTF.dqrts\n",
      "################ GTF, 1X-200-250, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R1-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R1-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R1-GTF.dqrts\n",
      "################ GTF, 1X-200-250, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R2-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R2-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R2-GTF.dqrts\n",
      "################ GTF, 1X-200-250, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R3-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R3-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R3-GTF.dqrts\n",
      "################ GTF, 1X-200-250, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R4-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R4-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R4-GTF.dqrts\n",
      "################ GTF, 1X-200-250, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R5-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R5-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R5-GTF.dqrts\n",
      "################ GTF, 1X-200-250, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R6-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R6-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R6-GTF.dqrts\n",
      "################ GTF, 1X-200-250, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R7-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R7-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R7-GTF.dqrts\n",
      "################ GTF, 1X-200-250, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R8-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R8-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R8-GTF.dqrts\n",
      "################ GTF, 1X-200-250, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R9-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R9-GTF.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R9-GTF.dqrts\n",
      "################ GTF, 1X-200-250, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R10-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R10-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R10-GTF.dqrts\n",
      "################ GTF, 1X-200-250, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R11-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R11-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R11-GTF.dqrts\n",
      "################ GTF, 1X-200-250, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R12-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R12-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R12-GTF.dqrts\n",
      "################ GTF, 1X-200-250, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R13-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R13-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R13-GTF.dqrts\n",
      "################ GTF, 1X-200-250, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R14-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R14-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R14-GTF.dqrts\n",
      "################ GTF, 1X-200-250, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R15-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R15-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R15-GTF.dqrts\n",
      "################ GTF, 1X-200-250, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R16-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R16-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R16-GTF.dqrts\n",
      "################ GTF, 1X-200-250, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R17-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R17-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R17-GTF.dqrts\n",
      "################ GTF, 1X-200-250, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R18-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R18-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R18-GTF.dqrts\n",
      "################ GTF, 1X-200-250, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R19-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R19-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R19-GTF.dqrts\n",
      "################ GTF, 1X-200-250, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R20-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R20-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R20-GTF.dqrts\n",
      "################ GTF, 1X-200-500, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R1-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R1-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R1-GTF.dqrts\n",
      "################ GTF, 1X-200-500, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R2-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R2-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R2-GTF.dqrts\n",
      "################ GTF, 1X-200-500, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R3-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R3-GTF.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R3-GTF.dqrts\n",
      "################ GTF, 1X-200-500, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R4-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R4-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R4-GTF.dqrts\n",
      "################ GTF, 1X-200-500, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R5-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R5-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R5-GTF.dqrts\n",
      "################ GTF, 1X-200-500, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R6-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R6-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R6-GTF.dqrts\n",
      "################ GTF, 1X-200-500, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R7-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R7-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R7-GTF.dqrts\n",
      "################ GTF, 1X-200-500, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R8-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R8-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R8-GTF.dqrts\n",
      "################ GTF, 1X-200-500, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R9-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R9-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R9-GTF.dqrts\n",
      "################ GTF, 1X-200-500, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R10-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R10-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R10-GTF.dqrts\n",
      "################ GTF, 1X-200-500, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R11-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R11-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R11-GTF.dqrts\n",
      "################ GTF, 1X-200-500, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R12-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R12-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R12-GTF.dqrts\n",
      "################ GTF, 1X-200-500, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R13-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R13-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R13-GTF.dqrts\n",
      "################ GTF, 1X-200-500, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R14-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R14-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R14-GTF.dqrts\n",
      "################ GTF, 1X-200-500, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R15-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R15-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R15-GTF.dqrts\n",
      "################ GTF, 1X-200-500, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R16-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R16-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R16-GTF.dqrts\n",
      "################ GTF, 1X-200-500, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R17-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R17-GTF.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R17-GTF.dqrts\n",
      "################ GTF, 1X-200-500, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R18-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R18-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R18-GTF.dqrts\n",
      "################ GTF, 1X-200-500, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R19-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R19-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R19-GTF.dqrts\n",
      "################ GTF, 1X-200-500, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R20-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R20-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R20-GTF.dqrts\n",
      "################ GTF, 1X-200-1000, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R1-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R1-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R1-GTF.dqrts\n",
      "################ GTF, 1X-200-1000, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R2-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R2-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R2-GTF.dqrts\n",
      "################ GTF, 1X-200-1000, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R3-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R3-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R3-GTF.dqrts\n",
      "################ GTF, 1X-200-1000, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R4-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R4-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R4-GTF.dqrts\n",
      "################ GTF, 1X-200-1000, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R5-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R5-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R5-GTF.dqrts\n",
      "################ GTF, 1X-200-1000, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R6-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R6-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R6-GTF.dqrts\n",
      "################ GTF, 1X-200-1000, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R7-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R7-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R7-GTF.dqrts\n",
      "################ GTF, 1X-200-1000, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R8-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R8-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R8-GTF.dqrts\n",
      "################ GTF, 1X-200-1000, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R9-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R9-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R9-GTF.dqrts\n",
      "################ GTF, 1X-200-1000, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R10-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R10-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R10-GTF.dqrts\n",
      "################ GTF, 1X-200-1000, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R11-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R11-GTF.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R11-GTF.dqrts\n",
      "################ GTF, 1X-200-1000, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R12-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R12-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R12-GTF.dqrts\n",
      "################ GTF, 1X-200-1000, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R13-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R13-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R13-GTF.dqrts\n",
      "################ GTF, 1X-200-1000, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R14-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R14-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R14-GTF.dqrts\n",
      "################ GTF, 1X-200-1000, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R15-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R15-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R15-GTF.dqrts\n",
      "################ GTF, 1X-200-1000, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R16-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R16-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R16-GTF.dqrts\n",
      "################ GTF, 1X-200-1000, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R17-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R17-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R17-GTF.dqrts\n",
      "################ GTF, 1X-200-1000, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R18-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R18-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R18-GTF.dqrts\n",
      "################ GTF, 1X-200-1000, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R19-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R19-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R19-GTF.dqrts\n",
      "################ GTF, 1X-200-1000, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R20-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R20-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R20-GTF.dqrts\n",
      "################ GTF, 0.5X-200-500, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R1-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R1-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R1-GTF.dqrts\n",
      "################ GTF, 0.5X-200-500, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R2-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R2-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R2-GTF.dqrts\n",
      "################ GTF, 0.5X-200-500, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R3-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R3-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R3-GTF.dqrts\n",
      "################ GTF, 0.5X-200-500, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R4-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R4-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R4-GTF.dqrts\n",
      "################ GTF, 0.5X-200-500, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R5-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R5-GTF.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R5-GTF.dqrts\n",
      "################ GTF, 0.5X-200-500, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R6-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R6-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R6-GTF.dqrts\n",
      "################ GTF, 0.5X-200-500, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R7-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R7-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R7-GTF.dqrts\n",
      "################ GTF, 0.5X-200-500, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R8-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R8-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R8-GTF.dqrts\n",
      "################ GTF, 0.5X-200-500, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R9-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R9-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R9-GTF.dqrts\n",
      "################ GTF, 0.5X-200-500, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R10-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R10-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R10-GTF.dqrts\n",
      "################ GTF, 0.5X-200-500, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R11-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R11-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R11-GTF.dqrts\n",
      "################ GTF, 0.5X-200-500, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R12-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R12-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R12-GTF.dqrts\n",
      "################ GTF, 0.5X-200-500, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R13-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R13-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R13-GTF.dqrts\n",
      "################ GTF, 0.5X-200-500, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R14-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R14-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R14-GTF.dqrts\n",
      "################ GTF, 0.5X-200-500, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R15-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R15-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R15-GTF.dqrts\n",
      "################ GTF, 0.5X-200-500, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R16-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R16-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R16-GTF.dqrts\n",
      "################ GTF, 0.5X-200-500, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R17-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R17-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R17-GTF.dqrts\n",
      "################ GTF, 0.5X-200-500, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R18-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R18-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R18-GTF.dqrts\n",
      "################ GTF, 0.5X-200-500, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R19-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R19-GTF.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R19-GTF.dqrts\n",
      "################ GTF, 0.5X-200-500, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R20-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R20-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R20-GTF.dqrts\n",
      "################ GTF, 2X-200-500, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R1-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R1-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R1-GTF.dqrts\n",
      "################ GTF, 2X-200-500, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R2-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R2-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R2-GTF.dqrts\n",
      "################ GTF, 2X-200-500, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R3-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R3-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R3-GTF.dqrts\n",
      "################ GTF, 2X-200-500, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R4-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R4-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R4-GTF.dqrts\n",
      "################ GTF, 2X-200-500, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R5-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R5-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R5-GTF.dqrts\n",
      "################ GTF, 2X-200-500, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R6-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R6-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R6-GTF.dqrts\n",
      "################ GTF, 2X-200-500, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R7-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R7-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R7-GTF.dqrts\n",
      "################ GTF, 2X-200-500, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R8-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R8-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R8-GTF.dqrts\n",
      "################ GTF, 2X-200-500, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R9-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R9-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R9-GTF.dqrts\n",
      "################ GTF, 2X-200-500, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R10-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R10-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R10-GTF.dqrts\n",
      "################ GTF, 2X-200-500, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R11-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R11-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R11-GTF.dqrts\n",
      "################ GTF, 2X-200-500, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R12-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R12-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R12-GTF.dqrts\n",
      "################ GTF, 2X-200-500, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R13-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R13-GTF.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R13-GTF.dqrts\n",
      "################ GTF, 2X-200-500, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R14-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R14-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R14-GTF.dqrts\n",
      "################ GTF, 2X-200-500, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R15-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R15-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R15-GTF.dqrts\n",
      "################ GTF, 2X-200-500, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R16-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R16-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R16-GTF.dqrts\n",
      "################ GTF, 2X-200-500, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R17-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R17-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R17-GTF.dqrts\n",
      "################ GTF, 2X-200-500, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R18-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R18-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R18-GTF.dqrts\n",
      "################ GTF, 2X-200-500, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R19-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R19-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R19-GTF.dqrts\n",
      "################ GTF, 2X-200-500, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R20-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R20-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R20-GTF.dqrts\n",
      "################ GTF, 1X-100-500, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R1-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R1-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R1-GTF.dqrts\n",
      "################ GTF, 1X-100-500, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R2-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R2-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R2-GTF.dqrts\n",
      "################ GTF, 1X-100-500, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R3-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R3-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R3-GTF.dqrts\n",
      "################ GTF, 1X-100-500, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R4-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R4-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R4-GTF.dqrts\n",
      "################ GTF, 1X-100-500, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R5-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R5-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R5-GTF.dqrts\n",
      "################ GTF, 1X-100-500, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R6-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R6-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R6-GTF.dqrts\n",
      "################ GTF, 1X-100-500, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R7-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R7-GTF.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R7-GTF.dqrts\n",
      "################ GTF, 1X-100-500, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R8-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R8-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R8-GTF.dqrts\n",
      "################ GTF, 1X-100-500, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R9-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R9-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R9-GTF.dqrts\n",
      "################ GTF, 1X-100-500, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R10-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R10-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R10-GTF.dqrts\n",
      "################ GTF, 1X-100-500, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R11-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R11-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R11-GTF.dqrts\n",
      "################ GTF, 1X-100-500, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R12-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R12-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R12-GTF.dqrts\n",
      "################ GTF, 1X-100-500, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R13-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R13-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R13-GTF.dqrts\n",
      "################ GTF, 1X-100-500, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R14-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R14-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R14-GTF.dqrts\n",
      "################ GTF, 1X-100-500, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R15-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R15-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R15-GTF.dqrts\n",
      "################ GTF, 1X-100-500, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R16-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R16-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R16-GTF.dqrts\n",
      "################ GTF, 1X-100-500, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R17-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R17-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R17-GTF.dqrts\n",
      "################ GTF, 1X-100-500, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R18-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R18-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R18-GTF.dqrts\n",
      "################ GTF, 1X-100-500, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R19-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R19-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R19-GTF.dqrts\n",
      "################ GTF, 1X-100-500, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R20-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R20-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R20-GTF.dqrts\n",
      "################ GTF, 1X-500-500, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R1-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R1-GTF.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R1-GTF.dqrts\n",
      "################ GTF, 1X-500-500, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R2-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R2-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R2-GTF.dqrts\n",
      "################ GTF, 1X-500-500, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R3-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R3-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R3-GTF.dqrts\n",
      "################ GTF, 1X-500-500, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R4-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R4-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R4-GTF.dqrts\n",
      "################ GTF, 1X-500-500, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R5-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R5-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R5-GTF.dqrts\n",
      "################ GTF, 1X-500-500, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R6-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R6-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R6-GTF.dqrts\n",
      "################ GTF, 1X-500-500, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R7-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R7-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R7-GTF.dqrts\n",
      "################ GTF, 1X-500-500, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R8-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R8-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R8-GTF.dqrts\n",
      "################ GTF, 1X-500-500, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R9-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R9-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R9-GTF.dqrts\n",
      "################ GTF, 1X-500-500, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R10-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R10-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R10-GTF.dqrts\n",
      "################ GTF, 1X-500-500, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R11-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R11-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R11-GTF.dqrts\n",
      "################ GTF, 1X-500-500, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R12-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R12-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R12-GTF.dqrts\n",
      "################ GTF, 1X-500-500, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R13-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R13-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R13-GTF.dqrts\n",
      "################ GTF, 1X-500-500, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R14-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R14-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R14-GTF.dqrts\n",
      "################ GTF, 1X-500-500, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R15-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R15-GTF.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R15-GTF.dqrts\n",
      "################ GTF, 1X-500-500, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R16-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R16-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R16-GTF.dqrts\n",
      "################ GTF, 1X-500-500, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R17-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R17-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R17-GTF.dqrts\n",
      "################ GTF, 1X-500-500, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R18-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R18-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R18-GTF.dqrts\n",
      "################ GTF, 1X-500-500, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R19-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R19-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R19-GTF.dqrts\n",
      "################ GTF, 1X-500-500, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R20-GTF.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R20-GTF.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R20-GTF.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-50, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R1-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R1-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R1-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-50, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R2-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R2-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R2-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-50, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R3-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R3-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R3-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-50, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R4-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R4-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R4-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-50, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R5-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R5-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R5-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-50, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R6-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R6-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R6-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-50, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R7-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R7-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R7-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-50, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R8-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R8-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R8-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-50, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R9-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R9-GTF-bucky-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R9-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-50, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R10-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R10-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R10-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-50, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R11-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R11-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R11-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-50, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R12-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R12-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R12-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-50, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R13-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R13-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R13-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-50, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R14-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R14-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R14-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-50, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R15-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R15-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R15-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-50, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R16-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R16-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R16-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-50, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R17-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R17-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R17-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-50, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R18-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R18-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R18-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-50, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R19-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R19-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R19-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-50, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R20-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R20-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R20-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-250, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R1-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R1-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R1-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-250, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R2-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R2-GTF-bucky-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R2-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-250, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R3-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R3-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R3-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-250, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R4-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R4-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R4-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-250, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R5-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R5-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R5-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-250, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R6-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R6-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R6-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-250, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R7-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R7-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R7-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-250, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R8-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R8-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R8-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-250, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R9-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R9-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R9-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-250, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R10-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R10-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R10-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-250, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R11-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R11-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R11-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-250, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R12-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R12-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R12-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-250, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R13-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R13-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R13-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-250, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R14-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R14-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R14-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-250, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R15-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R15-GTF-bucky-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R15-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-250, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R16-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R16-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R16-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-250, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R17-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R17-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R17-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-250, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R18-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R18-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R18-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-250, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R19-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R19-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R19-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-250, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R20-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R20-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R20-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-500, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R1-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R1-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R1-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-500, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R2-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R2-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R2-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-500, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R3-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R3-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R3-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-500, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R4-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R4-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R4-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-500, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R5-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R5-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R5-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-500, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R6-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R6-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R6-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-500, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R7-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R7-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R7-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-500, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R8-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R8-GTF-bucky-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R8-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-500, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R9-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R9-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R9-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-500, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R10-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R10-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R10-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-500, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R11-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R11-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R11-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-500, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R12-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R12-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R12-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-500, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R13-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R13-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R13-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-500, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R14-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R14-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R14-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-500, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R15-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R15-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R15-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-500, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R16-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R16-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R16-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-500, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R17-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R17-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R17-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-500, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R18-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R18-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R18-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-500, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R19-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R19-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R19-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-500, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R20-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R20-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R20-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-1000, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R1-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R1-GTF-bucky-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R1-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-1000, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R2-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R2-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R2-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-1000, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R3-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R3-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R3-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-1000, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R4-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R4-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R4-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-1000, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R5-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R5-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R5-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-1000, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R6-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R6-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R6-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-1000, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R7-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R7-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R7-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-1000, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R8-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R8-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R8-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-1000, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R9-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R9-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R9-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-1000, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R10-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R10-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R10-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-1000, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R11-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R11-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R11-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-1000, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R12-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R12-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R12-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-1000, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R13-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R13-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R13-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-1000, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R14-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R14-GTF-bucky-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R14-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-1000, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R15-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R15-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R15-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-1000, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R16-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R16-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R16-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-1000, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R17-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R17-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R17-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-1000, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R18-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R18-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R18-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-1000, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R19-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R19-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R19-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-200-1000, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R20-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R20-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R20-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 0.5X-200-500, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R1-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R1-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R1-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 0.5X-200-500, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R2-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R2-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R2-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 0.5X-200-500, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R3-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R3-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R3-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 0.5X-200-500, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R4-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R4-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R4-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 0.5X-200-500, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R5-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R5-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R5-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 0.5X-200-500, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R6-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R6-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R6-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 0.5X-200-500, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R7-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R7-GTF-bucky-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R7-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 0.5X-200-500, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R8-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R8-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R8-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 0.5X-200-500, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R9-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R9-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R9-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 0.5X-200-500, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R10-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R10-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R10-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 0.5X-200-500, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R11-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R11-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R11-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 0.5X-200-500, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R12-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R12-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R12-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 0.5X-200-500, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R13-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R13-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R13-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 0.5X-200-500, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R14-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R14-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R14-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 0.5X-200-500, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R15-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R15-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R15-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 0.5X-200-500, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R16-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R16-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R16-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 0.5X-200-500, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R17-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R17-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R17-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 0.5X-200-500, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R18-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R18-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R18-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 0.5X-200-500, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R19-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R19-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R19-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 0.5X-200-500, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R20-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R20-GTF-bucky-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R20-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 2X-200-500, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R1-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R1-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R1-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 2X-200-500, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R2-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R2-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R2-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 2X-200-500, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R3-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R3-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R3-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 2X-200-500, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R4-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R4-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R4-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 2X-200-500, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R5-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R5-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R5-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 2X-200-500, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R6-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R6-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R6-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 2X-200-500, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R7-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R7-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R7-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 2X-200-500, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R8-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R8-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R8-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 2X-200-500, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R9-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R9-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R9-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 2X-200-500, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R10-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R10-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R10-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 2X-200-500, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R11-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R11-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R11-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 2X-200-500, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R12-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R12-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R12-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 2X-200-500, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R13-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R13-GTF-bucky-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R13-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 2X-200-500, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R14-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R14-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R14-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 2X-200-500, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R15-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R15-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R15-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 2X-200-500, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R16-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R16-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R16-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 2X-200-500, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R17-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R17-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R17-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 2X-200-500, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R18-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R18-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R18-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 2X-200-500, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R19-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R19-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R19-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 2X-200-500, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R20-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R20-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R20-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-100-500, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R1-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R1-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R1-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-100-500, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R2-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R2-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R2-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-100-500, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R3-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R3-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R3-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-100-500, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R4-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R4-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R4-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-100-500, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R5-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R5-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R5-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-100-500, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R6-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R6-GTF-bucky-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R6-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-100-500, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R7-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R7-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R7-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-100-500, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R8-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R8-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R8-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-100-500, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R9-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R9-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R9-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-100-500, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R10-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R10-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R10-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-100-500, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R11-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R11-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R11-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-100-500, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R12-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R12-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R12-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-100-500, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R13-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R13-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R13-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-100-500, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R14-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R14-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R14-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-100-500, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R15-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R15-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R15-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-100-500, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R16-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R16-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R16-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-100-500, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R17-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R17-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R17-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-100-500, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R18-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R18-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R18-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-100-500, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R19-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R19-GTF-bucky-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R19-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-100-500, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R20-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R20-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R20-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-500-500, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R1-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R1-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R1-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-500-500, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R2-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R2-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R2-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-500-500, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R3-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R3-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R3-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-500-500, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R4-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R4-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R4-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-500-500, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R5-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R5-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R5-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-500-500, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R6-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R6-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R6-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-500-500, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R7-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R7-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R7-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-500-500, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R8-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R8-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R8-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-500-500, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R9-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R9-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R9-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-500-500, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R10-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R10-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R10-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-500-500, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R11-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R11-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R11-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-500-500, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R12-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R12-GTF-bucky-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R12-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-500-500, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R13-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R13-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R13-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-500-500, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R14-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R14-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R14-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-500-500, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R15-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R15-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R15-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-500-500, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R16-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R16-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R16-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-500-500, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R17-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R17-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R17-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-500-500, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R18-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R18-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R18-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-500-500, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R19-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R19-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R19-GTF-bucky-boot.dqrts\n",
      "################ GTF-bucky-boot, 1X-500-500, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R20-GTF-bucky-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R20-GTF-bucky-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R20-GTF-bucky-boot.dqrts\n",
      "################ GTF-boot, 1X-200-50, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R1-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R1-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R1-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-50, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R2-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R2-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R2-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-50, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R3-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R3-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R3-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-50, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R4-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R4-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R4-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-50, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R5-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R5-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R5-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-50, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R6-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R6-GTF-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R6-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-50, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R7-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R7-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R7-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-50, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R8-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R8-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R8-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-50, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R9-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R9-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R9-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-50, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R10-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R10-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R10-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-50, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R11-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R11-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R11-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-50, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R12-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R12-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R12-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-50, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R13-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R13-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R13-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-50, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R14-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R14-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R14-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-50, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R15-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R15-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R15-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-50, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R16-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R16-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R16-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-50, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R17-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R17-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R17-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-50, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R18-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R18-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R18-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-50, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R19-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R19-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R19-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-50, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R20-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R20-GTF-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-50/R20-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-250, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R1-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R1-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R1-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-250, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R2-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R2-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R2-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-250, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R3-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R3-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R3-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-250, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R4-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R4-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R4-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-250, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R5-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R5-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R5-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-250, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R6-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R6-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R6-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-250, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R7-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R7-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R7-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-250, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R8-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R8-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R8-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-250, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R9-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R9-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R9-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-250, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R10-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R10-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R10-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-250, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R11-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R11-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R11-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-250, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R12-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R12-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R12-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-250, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R13-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R13-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R13-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-250, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R14-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R14-GTF-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R14-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-250, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R15-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R15-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R15-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-250, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R16-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R16-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R16-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-250, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R17-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R17-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R17-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-250, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R18-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R18-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R18-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-250, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R19-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R19-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R19-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-250, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R20-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R20-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-250/R20-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-500, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R1-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R1-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R1-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-500, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R2-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R2-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R2-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-500, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R3-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R3-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R3-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-500, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R4-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R4-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R4-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-500, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R5-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R5-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R5-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-500, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R6-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R6-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R6-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-500, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R7-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R7-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R7-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-500, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R8-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R8-GTF-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R8-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-500, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R9-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R9-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R9-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-500, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R10-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R10-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R10-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-500, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R11-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R11-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R11-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-500, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R12-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R12-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R12-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-500, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R13-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R13-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R13-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-500, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R14-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R14-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R14-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-500, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R15-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R15-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R15-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-500, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R16-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R16-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R16-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-500, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R17-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R17-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R17-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-500, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R18-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R18-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R18-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-500, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R19-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R19-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R19-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-500, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R20-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R20-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-500/R20-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-1000, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R1-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R1-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R1-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-1000, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R2-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R2-GTF-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R2-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-1000, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R3-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R3-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R3-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-1000, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R4-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R4-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R4-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-1000, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R5-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R5-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R5-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-1000, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R6-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R6-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R6-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-1000, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R7-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R7-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R7-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-1000, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R8-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R8-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R8-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-1000, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R9-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R9-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R9-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-1000, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R10-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R10-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R10-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-1000, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R11-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R11-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R11-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-1000, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R12-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R12-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R12-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-1000, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R13-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R13-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R13-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-1000, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R14-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R14-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R14-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-1000, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R15-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R15-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R15-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-1000, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R16-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R16-GTF-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R16-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-1000, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R17-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R17-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R17-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-1000, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R18-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R18-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R18-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-1000, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R19-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R19-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R19-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-200-1000, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R20-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R20-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-200-1000/R20-GTF-boot.dqrts\n",
      "################ GTF-boot, 0.5X-200-500, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R1-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R1-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R1-GTF-boot.dqrts\n",
      "################ GTF-boot, 0.5X-200-500, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R2-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R2-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R2-GTF-boot.dqrts\n",
      "################ GTF-boot, 0.5X-200-500, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R3-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R3-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R3-GTF-boot.dqrts\n",
      "################ GTF-boot, 0.5X-200-500, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R4-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R4-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R4-GTF-boot.dqrts\n",
      "################ GTF-boot, 0.5X-200-500, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R5-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R5-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R5-GTF-boot.dqrts\n",
      "################ GTF-boot, 0.5X-200-500, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R6-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R6-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R6-GTF-boot.dqrts\n",
      "################ GTF-boot, 0.5X-200-500, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R7-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R7-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R7-GTF-boot.dqrts\n",
      "################ GTF-boot, 0.5X-200-500, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R8-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R8-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R8-GTF-boot.dqrts\n",
      "################ GTF-boot, 0.5X-200-500, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R9-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R9-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R9-GTF-boot.dqrts\n",
      "################ GTF-boot, 0.5X-200-500, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R10-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R10-GTF-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R10-GTF-boot.dqrts\n",
      "################ GTF-boot, 0.5X-200-500, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R11-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R11-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R11-GTF-boot.dqrts\n",
      "################ GTF-boot, 0.5X-200-500, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R12-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R12-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R12-GTF-boot.dqrts\n",
      "################ GTF-boot, 0.5X-200-500, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R13-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R13-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R13-GTF-boot.dqrts\n",
      "################ GTF-boot, 0.5X-200-500, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R14-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R14-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R14-GTF-boot.dqrts\n",
      "################ GTF-boot, 0.5X-200-500, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R15-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R15-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R15-GTF-boot.dqrts\n",
      "################ GTF-boot, 0.5X-200-500, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R16-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R16-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R16-GTF-boot.dqrts\n",
      "################ GTF-boot, 0.5X-200-500, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R17-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R17-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R17-GTF-boot.dqrts\n",
      "################ GTF-boot, 0.5X-200-500, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R18-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R18-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R18-GTF-boot.dqrts\n",
      "################ GTF-boot, 0.5X-200-500, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R19-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R19-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R19-GTF-boot.dqrts\n",
      "################ GTF-boot, 0.5X-200-500, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R20-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R20-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/0.5X-200-500/R20-GTF-boot.dqrts\n",
      "################ GTF-boot, 2X-200-500, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R1-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R1-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R1-GTF-boot.dqrts\n",
      "################ GTF-boot, 2X-200-500, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R2-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R2-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R2-GTF-boot.dqrts\n",
      "################ GTF-boot, 2X-200-500, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R3-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R3-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R3-GTF-boot.dqrts\n",
      "################ GTF-boot, 2X-200-500, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R4-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R4-GTF-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R4-GTF-boot.dqrts\n",
      "################ GTF-boot, 2X-200-500, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R5-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R5-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R5-GTF-boot.dqrts\n",
      "################ GTF-boot, 2X-200-500, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R6-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R6-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R6-GTF-boot.dqrts\n",
      "################ GTF-boot, 2X-200-500, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R7-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R7-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R7-GTF-boot.dqrts\n",
      "################ GTF-boot, 2X-200-500, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R8-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R8-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R8-GTF-boot.dqrts\n",
      "################ GTF-boot, 2X-200-500, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R9-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R9-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R9-GTF-boot.dqrts\n",
      "################ GTF-boot, 2X-200-500, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R10-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R10-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R10-GTF-boot.dqrts\n",
      "################ GTF-boot, 2X-200-500, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R11-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R11-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R11-GTF-boot.dqrts\n",
      "################ GTF-boot, 2X-200-500, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R12-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R12-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R12-GTF-boot.dqrts\n",
      "################ GTF-boot, 2X-200-500, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R13-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R13-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R13-GTF-boot.dqrts\n",
      "################ GTF-boot, 2X-200-500, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R14-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R14-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R14-GTF-boot.dqrts\n",
      "################ GTF-boot, 2X-200-500, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R15-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R15-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R15-GTF-boot.dqrts\n",
      "################ GTF-boot, 2X-200-500, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R16-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R16-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R16-GTF-boot.dqrts\n",
      "################ GTF-boot, 2X-200-500, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R17-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R17-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R17-GTF-boot.dqrts\n",
      "################ GTF-boot, 2X-200-500, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R18-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R18-GTF-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R18-GTF-boot.dqrts\n",
      "################ GTF-boot, 2X-200-500, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R19-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R19-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R19-GTF-boot.dqrts\n",
      "################ GTF-boot, 2X-200-500, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R20-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R20-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/2X-200-500/R20-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-100-500, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R1-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R1-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R1-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-100-500, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R2-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R2-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R2-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-100-500, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R3-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R3-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R3-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-100-500, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R4-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R4-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R4-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-100-500, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R5-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R5-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R5-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-100-500, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R6-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R6-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R6-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-100-500, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R7-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R7-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R7-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-100-500, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R8-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R8-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R8-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-100-500, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R9-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R9-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R9-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-100-500, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R10-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R10-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R10-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-100-500, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R11-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R11-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R11-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-100-500, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R12-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R12-GTF-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R12-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-100-500, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R13-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R13-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R13-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-100-500, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R14-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R14-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R14-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-100-500, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R15-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R15-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R15-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-100-500, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R16-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R16-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R16-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-100-500, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R17-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R17-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R17-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-100-500, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R18-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R18-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R18-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-100-500, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R19-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R19-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R19-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-100-500, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R20-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R20-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-100-500/R20-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-500-500, Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R1-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R1-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R1-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-500-500, Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R2-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R2-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R2-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-500-500, Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R3-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R3-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R3-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-500-500, Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R4-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R4-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R4-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-500-500, Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R5-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R5-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R5-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-500-500, Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R6-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R6-GTF-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R6-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-500-500, Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R7-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R7-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R7-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-500-500, Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R8-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R8-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R8-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-500-500, Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R9-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R9-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R9-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-500-500, Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R10-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R10-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R10-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-500-500, Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R11-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R11-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R11-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-500-500, Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R12-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R12-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R12-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-500-500, Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R13-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R13-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R13-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-500-500, Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R14-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R14-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R14-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-500-500, Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R15-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R15-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R15-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-500-500, Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R16-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R16-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R16-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-500-500, Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R17-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R17-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R17-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-500-500, Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R18-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R18-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R18-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-500-500, Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R19-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R19-GTF-boot.dqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R19-GTF-boot.dqrts\n",
      "################ GTF-boot, 1X-500-500, Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R20-GTF-boot.wqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R20-GTF-boot.dqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/37-taxon/Output/1X-500-500/R20-GTF-boot.dqrts\n"
     ]
    }
   ],
   "source": [
    "for input_wqrt in input_wqrts:\n",
    "    for configuration in configs:\n",
    "        if input_wqrt == \"GTF-true\":\n",
    "            continue\n",
    "#             tmp = configuration.split(\"-\")\n",
    "#             if len(tmp) > 2:\n",
    "#                 configuration = tmp[0] + \"-\" + tmp[1]\n",
    "         \n",
    "        output_folder = f'{base_folder}/15-taxon/Output/{configuration}'\n",
    "        GENES = int(configuration.split(\"gene\")[0])\n",
    "        for replicate_num in range(1, REPLICATES+1):\n",
    "            print(f\"################ {input_wqrt}, {configuration}, Replicate {replicate_num} ########################\")\n",
    "            input_wqrts_file = os.path.join(output_folder, f\"R{replicate_num}-{input_wqrt}.wqrts\")\n",
    "            cmd = f\"java -jar {dom_path} {input_wqrts_file} {get_dom_filename(input_wqrts_file)}\"\n",
    "            print(cmd)\n",
    "            os.system(cmd)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "9e2cfb58",
   "metadata": {},
   "outputs": [],
   "source": [
    "# for input_wqrt in input_wqrts:\n",
    "#     for configuration in configs:\n",
    "#         if input_wqrt == \"GTF-true\":\n",
    "#             tmp = configuration.split(\"-\")\n",
    "#             if len(tmp) > 2:\n",
    "#                 configuration = tmp[0] + \"-\" + tmp[1]\n",
    "         \n",
    "#         output_folder = f'{base_folder}/quartet_correction/Output/37-taxon/{configuration}'\n",
    "#         GENES = int(configuration.split(\"-\")[1])\n",
    "#         for replicate_num in range(1, REPLICATES+1):\n",
    "#             input_wqrts_file = os.path.join(output_folder, f\"R{replicate_num}-{input_wqrt}.wqrts\")\n",
    "#             if os.path.exists(input_wqrts_file) and not os.path.exists(get_dom_filename(input_wqrts_file)):\n",
    "#                 print(f\"Error in {input_wqrt}, {configuration}, {replicate_num}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "id": "5b18161c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-min2-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-min2-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-min-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-min2-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-min2-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-min2-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-min2-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-min2-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-min2-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-min2-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-min2-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-min-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-min2-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-min2-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-min2-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-min2-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-min2-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-min2-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-min2-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-min2-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-min-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-min2-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-min2-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-min2-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-min2-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-min2-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-min2-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-min2-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-min2-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-min-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-min2-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-min2-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-min2-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-min2-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-min2-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-min2-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-min2-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-min2-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-min-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-min2-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-min2-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-min2-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-min2-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-min2-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-min2-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-min2-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-min2-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-min-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-min2-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-min2-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-min2-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-min2-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-min2-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-min2-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-min2-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-min2-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-min-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-min2-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-min2-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-min2-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-min2-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-min2-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-min2-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-min2-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-min2-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-min-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-min2-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-min2-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-min2-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-min2-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-min2-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-min2-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-min2-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-min2-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-min-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-min2-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-min2-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-min2-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-min2-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-min2-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-min2-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-min2-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-min2-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-min-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-min2-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-min2-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-min2-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-min2-four.cdqrts\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Exception in thread \"main\" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1\n",
      "\tat BestWeightedQuartetGenerator.runner(BestWeightedQuartetGenerator.java:49)\n",
      "\tat BestWeightedQuartetGenerator.main(BestWeightedQuartetGenerator.java:26)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-min2-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-min2-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-min2-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-min2-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-min2-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-min2-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-min2-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-min2-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-min2-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-min2-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-min2-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-min2-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-min2-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-min2-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-min2-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-min2-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-min2-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-min2-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-min2-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-min2-four.cdqrts\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Exception in thread \"main\" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3\n",
      "\tat BestWeightedQuartetGenerator.runner(BestWeightedQuartetGenerator.java:51)\n",
      "\tat BestWeightedQuartetGenerator.main(BestWeightedQuartetGenerator.java:26)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-min2-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-min2-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-min2-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-min2-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-min2-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-min2-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-min2-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-min2-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-min2-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-min2-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-min2-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-min2-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-min2-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-min2-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-min2-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-min2-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-min2-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R1-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R1-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R1-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R1-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R1-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R1-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R1-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R1-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R1-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R1-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R1-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R1-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R1-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R1-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R1-GTF-min2-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R2-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R2-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R2-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R2-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R2-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R2-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R2-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R2-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R2-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R2-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R2-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R2-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R2-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R2-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R2-GTF-min2-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R3-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R3-GTF-min-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R3-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R3-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R3-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R3-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R3-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R3-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R3-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R3-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R3-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R3-GTF-min2-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R4-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R4-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R4-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R4-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R4-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R4-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R4-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R4-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R4-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R4-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R4-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R4-GTF-min2-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R5-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R5-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R5-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R5-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R5-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R5-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R5-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R5-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R5-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R5-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R5-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R5-GTF-min2-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R6-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R6-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R6-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R6-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R6-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R6-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R6-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R6-GTF-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R6-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R6-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R6-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R6-GTF-min2-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R7-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R7-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R7-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R7-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R7-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R7-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R7-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R7-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R7-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R7-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R7-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R7-GTF-min2-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R8-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R8-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R8-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R8-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R8-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R8-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R8-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R8-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R8-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R8-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R8-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R8-GTF-min2-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R9-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R9-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R9-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R9-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R9-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R9-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R9-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R9-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R9-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R9-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R9-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R9-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R9-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R9-GTF-min2-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R9-GTF-min2-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R10-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R10-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R10-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R10-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R10-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R10-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R10-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R10-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R10-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R10-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R10-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R10-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R10-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R10-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R10-GTF-min2-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R11-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R11-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R11-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R11-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R11-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R11-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R11-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R11-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R11-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R11-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R11-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R11-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R11-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R11-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R11-GTF-min2-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R12-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R12-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R12-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R12-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R12-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R12-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R12-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R12-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R12-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R12-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R12-GTF-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R12-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R12-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R12-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R12-GTF-min2-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R13-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R13-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R13-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R13-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R13-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R13-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R13-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R13-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R13-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R13-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R13-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R13-GTF-min2-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R14-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R14-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R14-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R14-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R14-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R14-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R14-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R14-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R14-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R14-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R14-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R14-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R14-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R14-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R14-GTF-min2-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R15-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R15-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R15-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R15-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R15-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R15-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R15-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R15-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R15-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R15-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R15-GTF-min2-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R15-GTF-min2-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R16-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R16-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R16-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R16-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R16-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R16-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R16-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R16-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R16-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R16-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R16-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R16-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R16-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R16-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R16-GTF-min2-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R17-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R17-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R17-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R17-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R17-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R17-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R17-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R17-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R17-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R17-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R17-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R17-GTF-min2-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R18-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R18-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R18-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R18-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R18-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R18-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R18-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R18-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R18-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R18-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R18-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R18-GTF-min2-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R19-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R19-GTF-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R19-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R19-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R19-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R19-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R19-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R19-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R19-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R19-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R19-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R19-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R19-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R19-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R19-GTF-min2-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R20-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R20-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R20-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R20-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R20-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R20-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R20-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R20-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R20-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R20-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R20-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200-500/R20-GTF-min2-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R1-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R1-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R1-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R1-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R1-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R1-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R1-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R1-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R1-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R1-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R1-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R1-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R1-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R1-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R1-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R1-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R1-GTF-min2-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R1-GTF-min2-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R2-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R2-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R2-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R2-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R2-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R2-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R2-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R2-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R2-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R2-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R2-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R2-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R2-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R2-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R2-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R2-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R2-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R2-GTF-min2-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R3-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R3-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R3-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R3-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R3-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R3-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R3-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R3-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R3-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R3-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R3-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R3-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R3-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R3-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R3-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R3-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R3-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R3-GTF-min2-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R4-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R4-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R4-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R4-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R4-GTF-min-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R4-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R4-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R4-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R4-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R4-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R4-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R4-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R4-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R4-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R4-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R4-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R4-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R4-GTF-min2-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R5-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R5-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R5-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R5-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R5-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R5-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R5-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R5-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R5-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R5-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R5-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R5-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R5-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R5-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R5-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R5-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R5-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R5-GTF-min2-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R6-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R6-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R6-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R6-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R6-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R6-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R6-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R6-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R6-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R6-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R6-GTF-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R6-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R6-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R6-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R6-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R6-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R6-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R6-GTF-min2-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R7-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R7-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R7-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R7-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R7-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R7-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R7-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R7-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R7-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R7-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R7-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R7-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R7-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R7-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R7-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R7-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R7-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R7-GTF-min2-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R8-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R8-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R8-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R8-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R8-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R8-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R8-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R8-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R8-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R8-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R8-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R8-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R8-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R8-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R8-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R8-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R8-GTF-min2-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R8-GTF-min2-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R9-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R9-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R9-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R9-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R9-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R9-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R9-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R9-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R9-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R9-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R9-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R9-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R9-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R9-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R9-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R9-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R9-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R9-GTF-min2-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R10-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R10-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R10-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R10-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R10-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R10-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R10-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R10-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R10-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R10-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R10-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R10-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R10-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R10-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R10-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R10-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R10-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R10-GTF-min2-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R11-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R11-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R11-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R11-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R11-GTF-min-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R11-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R11-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R11-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R11-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R11-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R11-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R11-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R11-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R11-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R11-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R11-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R11-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R11-GTF-min2-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R12-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R12-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R12-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R12-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R12-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R12-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R12-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R12-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R12-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R12-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R12-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R12-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R12-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R12-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R12-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R12-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R12-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R12-GTF-min2-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R13-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R13-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R13-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R13-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R13-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R13-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R13-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R13-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R13-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R13-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R13-GTF-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R13-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R13-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R13-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R13-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R13-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R13-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R13-GTF-min2-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R14-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R14-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R14-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R14-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R14-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R14-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R14-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R14-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R14-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R14-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R14-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R14-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R14-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R14-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R14-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R14-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R14-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R14-GTF-min2-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R15-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R15-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R15-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R15-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R15-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R15-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R15-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R15-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R15-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R15-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R15-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R15-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R15-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R15-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R15-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R15-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R15-GTF-min2-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R15-GTF-min2-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R16-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R16-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R16-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R16-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R16-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R16-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R16-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R16-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R16-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R16-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R16-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R16-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R16-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R16-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R16-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R16-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R16-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R16-GTF-min2-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R17-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R17-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R17-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R17-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R17-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R17-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R17-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R17-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R17-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R17-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R17-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R17-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R17-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R17-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R17-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R17-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R17-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R17-GTF-min2-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R18-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R18-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R18-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R18-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R18-GTF-min-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R18-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R18-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R18-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R18-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R18-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R18-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R18-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R18-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R18-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R18-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R18-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R18-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R18-GTF-min2-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R19-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R19-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R19-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R19-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R19-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R19-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R19-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R19-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R19-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R19-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R19-GTF-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R19-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R19-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R19-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R19-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R19-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R19-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R19-GTF-min2-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R20-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R20-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R20-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R20-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R20-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R20-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R20-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R20-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R20-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R20-GTF-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R20-GTF-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R20-GTF-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R20-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R20-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R20-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R20-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R20-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-400-50/R20-GTF-min2-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R1-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R1-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R1-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R1-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R1-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R1-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R1-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R1-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R1-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R1-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R1-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R1-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R1-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R1-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R1-GTF-min2-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R2-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R2-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R2-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R2-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R2-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R2-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R2-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R2-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R2-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R2-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R2-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R2-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R2-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R2-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R2-GTF-min2-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R3-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R3-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R3-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R3-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R3-GTF-min-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R3-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R3-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R3-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R3-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R3-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R3-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R3-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R3-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R3-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R3-GTF-min2-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R4-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R4-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R4-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R4-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R4-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R4-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R4-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R4-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R4-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R4-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R4-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R4-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R4-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R4-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R4-GTF-min2-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R5-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R5-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R5-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R5-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R5-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R5-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R5-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R5-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R5-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R5-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R5-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R5-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R5-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R5-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R5-GTF-min2-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R6-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R6-GTF-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R6-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R6-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R6-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R6-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R6-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R6-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R6-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R6-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R6-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R6-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R6-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R6-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R6-GTF-min2-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R7-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R7-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R7-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R7-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R7-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R7-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R7-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R7-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R7-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R7-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R7-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R7-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R7-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R7-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R7-GTF-min2-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R8-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R8-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R8-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R8-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R8-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R8-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R8-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R8-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R8-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R8-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R8-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R8-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R8-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R8-GTF-min2-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R8-GTF-min2-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R9-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R9-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R9-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R9-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R9-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R9-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R9-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R9-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R9-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R9-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R9-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R9-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R9-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R9-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R9-GTF-min2-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R10-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R10-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R10-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R10-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R10-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R10-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R10-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R10-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R10-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R10-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R10-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R10-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R10-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R10-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R10-GTF-min2-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R11-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R11-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R11-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R11-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R11-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R11-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R11-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R11-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R11-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R11-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R11-GTF-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R11-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R11-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R11-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R11-GTF-min2-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R12-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R12-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R12-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R12-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R12-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R12-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R12-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R12-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R12-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R12-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R12-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R12-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R12-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R12-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R12-GTF-min2-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R13-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R13-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R13-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R13-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R13-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R13-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R13-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R13-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R13-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R13-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R13-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R13-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R13-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R13-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R13-GTF-min2-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R14-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R14-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R14-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R14-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R14-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R14-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R14-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R14-GTF-min2-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R14-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R14-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R14-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R14-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R14-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R14-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R14-GTF-min2-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R15-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R15-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R15-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R15-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R15-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R15-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R15-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R15-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R15-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R15-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R15-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R15-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R15-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R15-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R15-GTF-min2-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R16-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R16-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R16-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R16-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R16-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R16-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R16-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R16-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R16-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R16-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R16-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R16-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R16-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R16-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R16-GTF-min2-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R17-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R17-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R17-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R17-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R17-GTF-min-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R17-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R17-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R17-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R17-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R17-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R17-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R17-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R17-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R17-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R17-GTF-min2-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R18-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R18-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R18-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R18-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R18-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R18-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R18-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R18-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R18-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R18-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R18-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R18-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R18-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R18-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R18-GTF-min2-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R19-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R19-GTF-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R19-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R19-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R19-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R19-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R19-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R19-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R19-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R19-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R19-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R19-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R19-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R19-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R19-GTF-min2-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R20-GTF-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R20-GTF-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R20-GTF-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R20-GTF-min-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R20-GTF-min-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R20-GTF-min-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R20-GTF-min2-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R20-GTF-min2-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R20-GTF-min2-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R20-GTF-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R20-GTF-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R20-GTF-min-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R20-GTF-min2-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R20-GTF-min2-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R20-GTF-min2-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R1-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R1-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R1-GTF-true-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R2-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R2-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R2-GTF-true-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R3-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R3-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R3-GTF-true-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R4-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R4-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R4-GTF-true-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R5-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R5-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R5-GTF-true-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R6-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R6-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R6-GTF-true-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R7-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R7-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R7-GTF-true-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R8-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R8-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R8-GTF-true-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R9-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R9-GTF-true-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R9-GTF-true-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R10-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R10-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R10-GTF-true-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R11-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R11-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R11-GTF-true-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R12-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R12-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R12-GTF-true-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R13-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R13-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R13-GTF-true-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R14-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R14-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R14-GTF-true-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R15-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R15-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R15-GTF-true-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R16-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R16-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R16-GTF-true-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R17-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R17-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R17-GTF-true-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R18-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R18-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R18-GTF-true-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R19-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R19-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R19-GTF-true-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R20-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R20-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R20-GTF-true-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R1-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R1-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R1-GTF-true-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R2-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R2-GTF-true-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R2-GTF-true-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R3-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R3-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R3-GTF-true-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R4-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R4-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R4-GTF-true-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R5-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R5-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R5-GTF-true-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R6-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R6-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R6-GTF-true-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R7-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R7-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R7-GTF-true-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R8-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R8-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R8-GTF-true-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R9-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R9-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R9-GTF-true-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R10-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R10-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R10-GTF-true-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R11-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R11-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R11-GTF-true-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R12-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R12-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R12-GTF-true-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R13-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R13-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R13-GTF-true-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R14-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R14-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R14-GTF-true-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R15-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R15-GTF-true-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R15-GTF-true-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R16-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R16-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R16-GTF-true-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R17-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R17-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R17-GTF-true-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R18-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R18-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R18-GTF-true-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R19-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R19-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R19-GTF-true-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R20-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R20-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R20-GTF-true-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R1-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R1-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R1-GTF-true-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R2-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R2-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R2-GTF-true-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R3-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R3-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R3-GTF-true-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R4-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R4-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R4-GTF-true-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R5-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R5-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R5-GTF-true-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R6-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R6-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R6-GTF-true-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R7-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R7-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R7-GTF-true-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R8-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R8-GTF-true-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R8-GTF-true-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R9-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R9-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R9-GTF-true-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R10-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R10-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R10-GTF-true-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R11-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R11-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R11-GTF-true-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R12-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R12-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R12-GTF-true-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R13-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R13-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R13-GTF-true-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R14-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R14-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R14-GTF-true-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R15-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R15-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R15-GTF-true-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R16-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R16-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R16-GTF-true-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R17-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R17-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R17-GTF-true-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R18-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R18-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R18-GTF-true-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R19-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R19-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R19-GTF-true-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R20-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R20-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R20-GTF-true-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R1-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R1-GTF-true-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R1-GTF-true-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R2-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R2-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R2-GTF-true-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R3-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R3-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R3-GTF-true-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R4-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R4-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R4-GTF-true-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R5-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R5-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R5-GTF-true-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R6-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R6-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R6-GTF-true-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R7-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R7-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R7-GTF-true-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R8-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R8-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R8-GTF-true-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R9-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R9-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R9-GTF-true-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R10-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R10-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R10-GTF-true-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R11-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R11-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R11-GTF-true-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R12-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R12-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R12-GTF-true-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R13-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R13-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R13-GTF-true-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R14-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R14-GTF-true-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R14-GTF-true-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R15-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R15-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R15-GTF-true-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R16-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R16-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R16-GTF-true-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R17-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R17-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R17-GTF-true-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R18-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R18-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R18-GTF-true-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R19-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R19-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R19-GTF-true-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R20-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R20-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200/R20-GTF-true-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R1-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R1-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R1-GTF-true-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R2-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R2-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R2-GTF-true-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R3-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R3-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R3-GTF-true-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R4-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R4-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R4-GTF-true-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R5-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R5-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R5-GTF-true-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R6-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R6-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R6-GTF-true-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R7-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R7-GTF-true-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R7-GTF-true-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R8-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R8-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R8-GTF-true-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R9-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R9-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R9-GTF-true-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R10-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R10-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R10-GTF-true-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R11-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R11-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R11-GTF-true-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R12-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R12-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R12-GTF-true-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R13-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R13-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R13-GTF-true-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R14-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R14-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R14-GTF-true-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R15-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R15-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R15-GTF-true-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R16-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R16-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R16-GTF-true-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R17-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R17-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R17-GTF-true-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R18-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R18-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R18-GTF-true-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R19-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R19-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R19-GTF-true-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R20-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R20-GTF-true-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200/R20-GTF-true-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R1-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R1-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R1-GTF-true-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R2-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R2-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R2-GTF-true-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R3-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R3-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R3-GTF-true-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R4-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R4-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R4-GTF-true-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R5-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R5-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R5-GTF-true-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R6-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R6-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R6-GTF-true-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R7-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R7-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R7-GTF-true-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R8-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R8-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R8-GTF-true-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R9-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R9-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R9-GTF-true-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R10-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R10-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R10-GTF-true-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R11-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R11-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R11-GTF-true-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R12-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R12-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R12-GTF-true-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R13-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R13-GTF-true-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R13-GTF-true-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R14-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R14-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R14-GTF-true-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R15-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R15-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R15-GTF-true-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R16-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R16-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R16-GTF-true-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R17-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R17-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R17-GTF-true-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R18-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R18-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R18-GTF-true-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R19-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R19-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R19-GTF-true-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R20-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R20-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/2X-200/R20-GTF-true-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "################ Replicate 2 ########################\n",
      "################ Replicate 3 ########################\n",
      "################ Replicate 4 ########################\n",
      "################ Replicate 5 ########################\n",
      "################ Replicate 6 ########################\n",
      "################ Replicate 7 ########################\n",
      "################ Replicate 8 ########################\n",
      "################ Replicate 9 ########################\n",
      "################ Replicate 10 ########################\n",
      "################ Replicate 11 ########################\n",
      "################ Replicate 12 ########################\n",
      "################ Replicate 13 ########################\n",
      "################ Replicate 14 ########################\n",
      "################ Replicate 15 ########################\n",
      "################ Replicate 16 ########################\n",
      "################ Replicate 17 ########################\n",
      "################ Replicate 18 ########################\n",
      "################ Replicate 19 ########################\n",
      "################ Replicate 20 ########################\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R1-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R1-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R1-GTF-true-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R2-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R2-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R2-GTF-true-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R3-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R3-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R3-GTF-true-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R4-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R4-GTF-true-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R4-GTF-true-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R5-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R5-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R5-GTF-true-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R6-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R6-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R6-GTF-true-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R7-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R7-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R7-GTF-true-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R8-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R8-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R8-GTF-true-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R9-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R9-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R9-GTF-true-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R10-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R10-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R10-GTF-true-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R11-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R11-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R11-GTF-true-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R12-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R12-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R12-GTF-true-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R13-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R13-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R13-GTF-true-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R14-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R14-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R14-GTF-true-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R15-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R15-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R15-GTF-true-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R16-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R16-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R16-GTF-true-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R17-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R17-GTF-true-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R17-GTF-true-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R18-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R18-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R18-GTF-true-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R19-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R19-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R19-GTF-true-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R20-GTF-true-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R20-GTF-true-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500/R20-GTF-true-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-bucky-boot-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-bucky-boot-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-bucky-boot-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-bucky-boot-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-bucky-boot-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-bucky-boot-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-bucky-boot-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-bucky-boot-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-bucky-boot-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-bucky-boot-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-bucky-boot-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-bucky-boot-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-bucky-boot-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-bucky-boot-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-bucky-boot-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-bucky-boot-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-bucky-boot-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-bucky-boot-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-bucky-boot-four.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-bucky-boot-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-bucky-boot-four.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-bucky-boot-four.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-bucky-boot-four.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-bucky-boot-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-bucky-boot-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-bucky-boot-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "################ Replicate 2 ########################\n",
      "################ Replicate 3 ########################\n",
      "################ Replicate 4 ########################\n",
      "################ Replicate 5 ########################\n",
      "################ Replicate 6 ########################\n",
      "################ Replicate 7 ########################\n",
      "################ Replicate 8 ########################\n",
      "################ Replicate 9 ########################\n",
      "################ Replicate 10 ########################\n",
      "################ Replicate 11 ########################\n",
      "################ Replicate 12 ########################\n",
      "################ Replicate 13 ########################\n",
      "################ Replicate 14 ########################\n",
      "################ Replicate 15 ########################\n",
      "################ Replicate 16 ########################\n",
      "################ Replicate 17 ########################\n",
      "################ Replicate 18 ########################\n",
      "################ Replicate 19 ########################\n",
      "################ Replicate 20 ########################\n",
      "################ Replicate 1 ########################\n",
      "################ Replicate 2 ########################\n",
      "################ Replicate 3 ########################\n",
      "################ Replicate 4 ########################\n",
      "################ Replicate 5 ########################\n",
      "################ Replicate 6 ########################\n",
      "################ Replicate 7 ########################\n",
      "################ Replicate 8 ########################\n",
      "################ Replicate 9 ########################\n",
      "################ Replicate 10 ########################\n",
      "################ Replicate 11 ########################\n",
      "################ Replicate 12 ########################\n",
      "################ Replicate 13 ########################\n",
      "################ Replicate 14 ########################\n",
      "################ Replicate 15 ########################\n",
      "################ Replicate 16 ########################\n",
      "################ Replicate 17 ########################\n",
      "################ Replicate 18 ########################\n",
      "################ Replicate 19 ########################\n",
      "################ Replicate 20 ########################\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R1-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R1-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R1-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R2-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R2-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R2-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R3-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R3-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R3-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R4-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R4-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R4-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R5-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R5-GTF-bucky-boot-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R5-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R6-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R6-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R6-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R7-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R7-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R7-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R8-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R8-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R8-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R9-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R9-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R9-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R10-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R10-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R10-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R11-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R11-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R11-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R12-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R12-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R12-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R13-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R13-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R13-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R14-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R14-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R14-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R15-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R15-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R15-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R16-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R16-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R16-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R17-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R17-GTF-bucky-boot-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R17-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R18-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R18-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R18-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R19-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R19-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R19-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R20-GTF-bucky-boot-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R20-GTF-bucky-boot-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-500-500/R20-GTF-bucky-boot-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-a1-s500-bucky-mrbayes-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-a1-s500-bucky-mrbayes-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-a1-s500-bucky-mrbayes-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-a1-s500-bucky-mrbayes-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-a1-s500-bucky-mrbayes-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-a1-s500-bucky-mrbayes-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-a1-s500-bucky-mrbayes-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-a1-s500-bucky-mrbayes-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-a1-s500-bucky-mrbayes-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-a1-s500-bucky-mrbayes-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "################ Replicate 2 ########################\n",
      "################ Replicate 3 ########################\n",
      "################ Replicate 4 ########################\n",
      "################ Replicate 5 ########################\n",
      "################ Replicate 6 ########################\n",
      "################ Replicate 7 ########################\n",
      "################ Replicate 8 ########################\n",
      "################ Replicate 9 ########################\n",
      "################ Replicate 10 ########################\n",
      "################ Replicate 11 ########################\n",
      "################ Replicate 12 ########################\n",
      "################ Replicate 13 ########################\n",
      "################ Replicate 14 ########################\n",
      "################ Replicate 15 ########################\n",
      "################ Replicate 16 ########################\n",
      "################ Replicate 17 ########################\n",
      "################ Replicate 18 ########################\n",
      "################ Replicate 19 ########################\n",
      "################ Replicate 20 ########################\n",
      "################ Replicate 1 ########################\n",
      "################ Replicate 2 ########################\n",
      "################ Replicate 3 ########################\n",
      "################ Replicate 4 ########################\n",
      "################ Replicate 5 ########################\n",
      "################ Replicate 6 ########################\n",
      "################ Replicate 7 ########################\n",
      "################ Replicate 8 ########################\n",
      "################ Replicate 9 ########################\n",
      "################ Replicate 10 ########################\n",
      "################ Replicate 11 ########################\n",
      "################ Replicate 12 ########################\n",
      "################ Replicate 13 ########################\n",
      "################ Replicate 14 ########################\n",
      "################ Replicate 15 ########################\n",
      "################ Replicate 16 ########################\n",
      "################ Replicate 17 ########################\n",
      "################ Replicate 18 ########################\n",
      "################ Replicate 19 ########################\n",
      "################ Replicate 20 ########################\n",
      "################ Replicate 1 ########################\n",
      "################ Replicate 2 ########################\n",
      "################ Replicate 3 ########################\n",
      "################ Replicate 4 ########################\n",
      "################ Replicate 5 ########################\n",
      "################ Replicate 6 ########################\n",
      "################ Replicate 7 ########################\n",
      "################ Replicate 8 ########################\n",
      "################ Replicate 9 ########################\n",
      "################ Replicate 10 ########################\n",
      "################ Replicate 11 ########################\n",
      "################ Replicate 12 ########################\n",
      "################ Replicate 13 ########################\n",
      "################ Replicate 14 ########################\n",
      "################ Replicate 15 ########################\n",
      "################ Replicate 16 ########################\n",
      "################ Replicate 17 ########################\n",
      "################ Replicate 18 ########################\n",
      "################ Replicate 19 ########################\n",
      "################ Replicate 20 ########################\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-SVD-reciprocal-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R1-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R2-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R3-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R4-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R5-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R6-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-SVD-reciprocal-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R7-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R8-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R9-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R10-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R11-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R12-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R13-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-SVD-reciprocal-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R14-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R15-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R16-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R17-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R18-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R19-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-SVD-reciprocal-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-50/R20-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R1-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R2-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R3-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R4-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R5-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R6-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-SVD-reciprocal-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R7-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R8-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R9-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R10-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R11-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R12-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-SVD-reciprocal-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R13-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R14-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R15-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R16-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R17-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R18-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R19-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-SVD-reciprocal-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-250/R20-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R1-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R2-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R3-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R4-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R5-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-SVD-reciprocal-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R6-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R7-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R8-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R9-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R10-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R11-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R12-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-SVD-reciprocal-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R13-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R14-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R15-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R16-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R17-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R18-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-SVD-reciprocal-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R19-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-500/R20-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R1-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R2-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R3-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R4-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R5-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-SVD-reciprocal-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R6-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R7-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R8-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R9-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R10-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R11-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-SVD-reciprocal-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R12-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R13-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R14-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R15-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R16-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R17-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R18-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-SVD-reciprocal-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R19-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/1X-200-1000/R20-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R1-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 2 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R2-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 3 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R3-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 4 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R4-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 5 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-SVD-reciprocal-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R5-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 6 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R6-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 7 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R7-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 8 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R8-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 9 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R9-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 10 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R10-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 11 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R11-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 12 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-SVD-reciprocal-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R12-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 13 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R13-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 14 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R14-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 15 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R15-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 16 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R16-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 17 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R17-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 18 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-SVD-reciprocal-min-six.cdqrts\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R18-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 19 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R19-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 20 ########################\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-SVD-reciprocal-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-SVD-reciprocal-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-SVD-reciprocal-six.cdqrts\n",
      "java -jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/Scripts-2/generateBestWQrts.jar /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-SVD-reciprocal-min-six.cwqrts /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-SVD-reciprocal-min-six.cdqrts\n",
      ">>> Done generating best weighted quartets to outputFile = /home/navid/comparative_study/Alignments-WeightedQuartets-SpeciesTree-main/quartet_correction/Output/37-taxon/0.5X-200-500/R20-SVD-reciprocal-min-six.cdqrts\n",
      "################ Replicate 1 ########################\n",
      "################ Replicate 2 ########################\n",
      "################ Replicate 3 ########################\n",
      "################ Replicate 4 ########################\n",
      "################ Replicate 5 ########################\n",
      "################ Replicate 6 ########################\n",
      "################ Replicate 7 ########################\n",
      "################ Replicate 8 ########################\n",
      "################ Replicate 9 ########################\n",
      "################ Replicate 10 ########################\n",
      "################ Replicate 11 ########################\n",
      "################ Replicate 12 ########################\n",
      "################ Replicate 13 ########################\n",
      "################ Replicate 14 ########################\n",
      "################ Replicate 15 ########################\n",
      "################ Replicate 16 ########################\n",
      "################ Replicate 17 ########################\n",
      "################ Replicate 18 ########################\n",
      "################ Replicate 19 ########################\n",
      "################ Replicate 20 ########################\n",
      "################ Replicate 1 ########################\n",
      "################ Replicate 2 ########################\n",
      "################ Replicate 3 ########################\n",
      "################ Replicate 4 ########################\n",
      "################ Replicate 5 ########################\n",
      "################ Replicate 6 ########################\n",
      "################ Replicate 7 ########################\n",
      "################ Replicate 8 ########################\n",
      "################ Replicate 9 ########################\n",
      "################ Replicate 10 ########################\n",
      "################ Replicate 11 ########################\n",
      "################ Replicate 12 ########################\n",
      "################ Replicate 13 ########################\n",
      "################ Replicate 14 ########################\n",
      "################ Replicate 15 ########################\n",
      "################ Replicate 16 ########################\n",
      "################ Replicate 17 ########################\n",
      "################ Replicate 18 ########################\n",
      "################ Replicate 19 ########################\n",
      "################ Replicate 20 ########################\n",
      "################ Replicate 1 ########################\n",
      "################ Replicate 2 ########################\n",
      "################ Replicate 3 ########################\n",
      "################ Replicate 4 ########################\n",
      "################ Replicate 5 ########################\n",
      "################ Replicate 6 ########################\n",
      "################ Replicate 7 ########################\n",
      "################ Replicate 8 ########################\n",
      "################ Replicate 9 ########################\n",
      "################ Replicate 10 ########################\n",
      "################ Replicate 11 ########################\n",
      "################ Replicate 12 ########################\n",
      "################ Replicate 13 ########################\n",
      "################ Replicate 14 ########################\n",
      "################ Replicate 15 ########################\n",
      "################ Replicate 16 ########################\n",
      "################ Replicate 17 ########################\n",
      "################ Replicate 18 ########################\n",
      "################ Replicate 19 ########################\n",
      "################ Replicate 20 ########################\n"
     ]
    }
   ],
   "source": [
    "# corrected ones\n",
    "for input_wqrt in input_wqrts:\n",
    "    for configuration in configs:\n",
    "        if input_wqrt == \"GTF-true\":\n",
    "            tmp = configuration.split(\"-\")\n",
    "            if len(tmp) > 2:\n",
    "                configuration = tmp[0] + \"-\" + tmp[1]\n",
    "                \n",
    "        output_folder = f'{base_folder}/quartet_correction/Output/37-taxon/{configuration}'\n",
    "        \n",
    "        GENES = int(configuration.split(\"-\")[1])\n",
    "        \n",
    "        for replicate_num in range(1, REPLICATES+1):\n",
    "            print(f\"################ Replicate {replicate_num} ########################\")\n",
    "            for mode in modes:\n",
    "                for weighting in weights:\n",
    "                    if weighting == \"regular\":\n",
    "                        input_wqrts_file = os.path.join(output_folder, f\"R{replicate_num}-{input_wqrt}-{mode}.cwqrts\")\n",
    "                    else:\n",
    "                        input_wqrts_file = os.path.join(output_folder, f\"R{replicate_num}-{input_wqrt}-{weighting}-{mode}.cwqrts\")\n",
    "\n",
    "                    if os.path.exists(input_wqrts_file):\n",
    "                        cmd = f\"java -jar {dom_path} {input_wqrts_file} {get_dom_filename(input_wqrts_file)}\"\n",
    "                        print(cmd)\n",
    "                        os.system(cmd)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "2ab61580",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
